### DISCUSSION TITLE
wow

### DISCUSSION THREAD

**Discussion by akbar-ardiansyah** [community/community#18339] [Other Feature Feedback, Questions, & Ideas]
I get it shark badges

**Comment by ale-lit** [comment]
Congratulations )

Here you can learn more about it: https://github.com/Schweinepriester/github-profile-achievements

**Reply by artabed** [reply]
Congratulation 👏👏👏

**Reply by miteokaku** [reply]
👍🏿

**Comment by mushonnip** [comment]
nice, i have it too...
now i'm looking for galaxy brain
